index.js

```diff
@@ -50,6 +51,19 @@ module.exports = {
         }
 
         if (route.handler && route.path && route.method) {
+          const originalHandler = route.handler;
+          route.handler = async (request, h) => {
+            try {
+              return await originalHandler.call(null, request, h);
+            } catch (error) {
+              if (options.errorHandler) {
+                await options.errorHandler(request, error);
+              } else {
+                server.log(['hapi-plus-routes', request.method, request.path], 'No error handler defined');
```

Comment:
We dont want to log that. In case project has not applied global error handler, just throw is enough.